backend/editor/entries.py

```diff
@@ -87,7 +87,8 @@ async def parse_taxonomy(self, filename):
         Helper function to call the Open Food Facts Python Taxonomy Parser
         """
         # Close current transaction to use the session variable in parser
-        await get_current_transaction().commit()
+        async with get_current_transaction() as transaction:
+            transaction.commit()
```

Comment:
hum, it seems strange to me that you have to do this ! (but it could be, just I don't have the whole picture in my head now...)

Maybe put a `# TODO: verify if really needed after parser refactor`

Reply:
you're right, while working on handling background task on frontend, I realize I shouldn't do this.